resources/default.rb

```diff
@@ -4,19 +4,59 @@
 property :artifactoryonline, :kind_of => String
 property :repository, :kind_of => String, :required => true
 property :repository_path, :kind_of => String, :required => true
+property :artifact_name, :kind_of => String, :required => true
 property :artifactory_username, :kind_of => String
 property :artifactory_password, :kind_of => String
+property :highest, :kind_of => [TrueClass, FalseClass]
```

Comment:
As a property name, I _feel_ giving some more descriptive name like "fetch_highest" or "fetch_latest" could be better.